volumes/miovision/README.md

```diff
@@ -143,7 +143,7 @@ intersection_uid|integer|Identifier linking to specific intersection stored in `
 datetime_bin|timestamp without time zone|Start of 1-minute time bin in EDT|2017-10-13 09:07:00|
 classification_uid|text|Identifier linking to specific mode class stored in `classifications`|1|
 leg|text|Entry leg of movement|E|
-movement_uid|integer|Identifier linking to specific turning movement stored in `movements`|2|
+movement_uid|integer|Identifier linking to specific turning movement stored in `movement_map`|2|
```

Comment:
I just created `miovision_api.movements` to store the lookup for API movements. Did not really realise these weren't available before

Reply:
Ahh, `movements` should also be documented in the `README.md`.